docs/howto/operate/setup-ci-cd-aws.md

```diff
@@ -0,0 +1,145 @@
+(operate:setup-ci-cd-aws)=
+# Set up the CI/CD system in an AWS cluster
```

Comment:
Since we might want to set up CI/CD auto-deploys across all of the cloud providers, perhaps we should create a top-level section under `Topics` called `Automatic deployment with CI/CD`, and then underneath that are pages for each of the major cloud providers. (this assumes that it will be slightly different for AWS, GCP, and Azure).

Does that make sense to you?